infra/app/api.bicep

```diff
@@ -25,6 +32,10 @@ module api '../core/host/appservice.bicep' = {
     runtimeName: 'dotnetcore'
     runtimeVersion: '6.0'
     scmDoBuildDuringDeployment: false
+    targetResourceId: targetResourceId
+    appUser: appUser
+    appUserPassword: appUserPassword
```

Comment:
Why does appservice need to know about appUser?  We don't want to couple the two at the appservice level.

Reply:
appservice doesn't know appUser, it's a param passed in from main, user knows their app, and their db's username/password.